Doc/library/http.server.rst

```diff
@@ -502,11 +502,22 @@ following command runs an HTTP/1.1 conformant server::
    Note that CGI scripts will be run with UID of user nobody, for security
    reasons.  Problems with the CGI script will be translated to error 403.
 
+   .. deprecated-removed:: 3.13 3.15
+   :class:`CGIHTTPRequestHandler` is being removed in 3.15.  CGI has not been
+   considered a good way to do things for well over a decade, this code has
+   been unmaintained for a while now and sees very little practical use.
+   Retaining it could lead to further
+   :ref:`security considerations <http.server-security>`.
```

Comment:
```suggestion
   .. deprecated-removed:: 3.13 3.15

      :class:`CGIHTTPRequestHandler` is being removed in 3.15.  CGI has not been
      considered a good way to do things for well over a decade, this code has
      been unmaintained for a while now and sees very little practical use.
      Retaining it could lead to further
      :ref:`security considerations <http.server-security>`.
```